components/ActionBar/ActionBar.tsx

```diff
@@ -1,15 +1,12 @@
 import React from 'react'
 import { ButtonAnchor } from 'components/global/Button/Button'
-import { PageMetadata } from 'components/global/withPageMetadata'
 import { StyledActionBarContainer } from './ActionBar.styled'
 import getConferenceActions from 'config/actions'
+import { useConfig } from 'Context/Config'
 
-interface ActionBarProps {
```

Comment:
Changes like this are where the pagemetadata is removed and replace with the hook